runtime/compiler/z/codegen/S390PrivateLinkage.cpp

```diff
@@ -1583,7 +1583,11 @@ J9::Z::PrivateLinkage::buildNoPatchingVirtualDispatchWithResolve(TR::Node *callN
       int32_t vtableOffset;
       };
    
-   ccResolveVirtualData *ccResolveVirtualDataAddress = reinterpret_cast<ccResolveVirtualData *>(cg()->allocateCodeMemory(sizeof(ccResolveVirtualData), false));
+   OMR::CCData *codeCacheData = cg()->getCodeCache()->manager()->getCodeCacheData();
+   OMR::CCData::index_t index;
+   codeCacheData->put(NULL, sizeof(ccResolveVirtualData), 8, NULL, index);
+   
+   ccResolveVirtualData *ccResolveVirtualDataAddress = codeCacheData->get<ccResolveVirtualData>(index);
```

Comment:
Likewise here.

Reply:
Fixed in https://github.com/eclipse/openj9/compare/e90116d7332681724cdd8aa492ff4522dadd0d6a..69c427858ff45f769e7f9831d4e4777159a0a758

Reply:
Checking result of `put` in https://github.com/eclipse/openj9/compare/69c427858ff45f769e7f9831d4e4777159a0a758..e6572d23ab0cf110a98cc239bea656e8aa8729ec